collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -286,7 +276,7 @@ private void clearChoices() {
     @Override
     protected void onListItemClick(ListView l, View v, int position, long id) {
         super.onListItemClick(l, v, position, id);
-        mDownloadButton.setEnabled(!(selectedItemCount() == 0));
+        mDownloadButton.setEnabled(!(ListViewUtils.selectedItemCount(getListView()) == 0));
```

Comment:
I think using != would make this easier to think about. Actually, I think > 0 makes even more sense!